src/styles/container.scss

```diff
@@ -0,0 +1,33 @@
+.container {
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: $text-color;
+
+  @each $number, $color in $colors {
+    :nth-child(#{$number}) {
+      background-color: $color;
+    }
+  }
+
+  &__block {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    height: $block-size;
+    min-width: $block-size;
+  }
+
+  @media (min-width: 600px) {
+    & {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+    }
+  }
```

Comment:
1. No need to use & in media when using SASS. Just write it inside correct selector.
```
@media (min-width: 600px) {
   ...styles
}
```
2. Set `display: grid` as default style.